types/parse-prefer-header/parse-prefer-header-tests.ts

```diff
@@ -1,4 +1,6 @@
 import parsePreferHeader = require('parse-prefer-header');
 
-parsePreferHeader(['respond-async, wait=100', 'handling=lenient']);
+parsePreferHeader(['respond-async, wait=100', 'handling=lenient'] as ReadonlyArray<string>);
```

Comment:
Oh look, you just did what I wrote in the previous comment :)

Reply:
This is for the sake of testing whether it accepts a readonly one. Compiles just as fine with the original code.